api/v1beta1/openstackmachine_types.go

```diff
@@ -43,7 +43,7 @@ type OpenStackMachineSpec struct {
 
 	// The image to use for your server instance.
 	// If the rootVolume is specified, this will be used when creating the root volume.
-	Image ImageFilter `json:"image,omitempty"`
+	Image ImageFilter `json:"image"`
```

Comment:
Please can you add an explicit `// +optional`?

Reply:
I wasn't exactly sure if the image is optional. Is it? Should I make it `Image *ImageFilter` then?

Reply:
I think we can't make `ImageFilter` optional, code requires that the exactly one image will be found. An empty `ImageFilter` might be possible and resolvable if you have just one image in that tenant.

Reply:
D'oh. Of course this should not be optional. Yep, lets make it explicitly required.